util/assetbuilder/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM flynn/cedarish
+
+ADD Gemfile /app/Gemfile
+ADD Gemfile.lock /app/Gemfile.lock
+
+# install Go
+RUN wget https://storage.googleapis.com/golang/go1.4.2.linux-amd64.tar.gz
+RUN tar -C /usr/local -zxvf go1.4.2.linux-amd64.tar.gz
+ENV PATH /usr/local/go/bin:$PATH
+ENV GOPATH /build/gopath
```

Comment:
Can we do the build outside of the container via Tup? We already have a Go toolchain available, so this just slows things down.